stacks-signer/release-process.md

```diff
@@ -11,75 +11,82 @@
 | Linux ARMv7                 | _builds are provided but not tested_ |
 | Linux ARM64                 | _builds are provided but not tested_ |
 
-
 ## Release Schedule and Hotfixes
 
-Normal releases in this repository that add new or updated features shall be released in an ad-hoc manner. The currently staged changes for such releases
-are in the [develop branch](https://github.com/stacks-network/stacks-core/tree/develop). It is generally safe to run a `stacks-signer` from that branch, though it has received less rigorous testing than release branches. If bugs are found in the `develop` branch, please do [report them as issues](https://github.com/stacks-network/stacks-core/issues) on this repository.
+Normal releases in this repository that add new or updated features shall be released in an ad-hoc manner.
+The currently staged changes for such releases are in the [develop branch](https://github.com/stacks-network/stacks-core/tree/develop).
```

Comment:
These lines confused me because `docs/release-process` mention a monthly release schedule.

Also, "this repository" here refers to both `stacks-core` and `stacks-signer`. Maybe just:

```suggestion
`stack-signer` releases that add new or updated features shall be released in an ad-hoc manner.
```